src/clj/harja/palvelin/integraatiot/api/ilmoitukset.clj

```diff
@@ -154,6 +156,37 @@
                                          (when kuuntelun-lopetus-fn
                                            (kuuntelun-lopetus-fn)))))))))))))
 
+(defn hae-kovakoodatut-ilmoitukset-ytunnuksella [db parametrit kayttaja]
+  (let [;; Varmista, että annettu y-tunnus kuuluu käyttäjälle
+        ytunnus-loytyy? (:exists (first (kayttajat-kyselyt/kayttajan-organisaation-ytunnus-loytyy
+                                          db {:kayttajanimi (:kayttajanimi kayttaja)
+                                              :ytunnus (:ytunnus parametrit)})))
+        _ (when-not ytunnus-loytyy?
+            (throw+ {:type virheet/+viallinen-kutsu+
+                     :virheet [{:koodi virheet/+tuntematon-kayttaja-koodi+
+                                :viesti "Käyttäjä ja y-tunnus ei täsmää."}]}))
+        ;; Muodostetaan kovakoodattu vastaus.
+        ilmoitukset {:ilmoitukset [{:ilmoitus {:ilmoitusid 123
+                                               :tunniste "UV-1509-1a"
+                                               :tila "vastaanotettu"
+                                               :ilmoitettu "2015-09-29T14:49:45"
+                                               :valitetty-harjaan "2015-09-29T14:50:45"
+                                               :valitetty-urakkaan "2015-09-29T14:49:45"
+                                               :vastaanotettu-harjaan "2015-09-29T14:49:45"
+                                               :paivitetty-harjaan nil
+                                               :ilmoitustyyppi "toimenpidepyynto",
+                                               :yhteydenottopyynto true,
+                                               :ilmoittaja {:sukunimi "Porinmatti", :etunimi "Pekka", :matkapuhelin "0502234567", :tyopuhelin "0501234567", :email "tyonvalvonta@example.org"}
+                                               :lahettaja {:etunimi "Ilmari", :sukunimi "Ilmoitus", :matkapuhelin nil, :tyopuhelin nil, :email "ilmari.ilmoitus@example.org"},
+                                               :selitteet [{:selite "tiellaOnEste"}],
+                                               :sijainti {:koordinaatit {:x 430716.0, :y 7200111.0}},
+                                               :tienumero nil,
+                                               :otsikko nil,
+                                               :paikankuvaus "Pasku lumipeite nelostiellä Ristisuon kohdalla ja tiet auraamatta.",
+                                               :lisatieto nil,
+                                               :aiheutti-toimenpiteita nil,}}]}]
```

Comment:
Tänne vielä mukaan ilmoitukseen liittyvät kuittaukset ilmoitustoimenpide-taulusta.

Reply:
Lisään nämä, kunhan uudistuneen scheman saan tähän matkaan. Erittäin hyvä bongaus!